admin/brand.php

```diff
@@ -53,98 +56,36 @@
-//     $Brand_Desc = $_POST['Brand_Desc'];    
-        
-//     // check empty fields
-//     if(empty($Brand_ID) || empty($Brand_Name) || empty($Brand_Desc)) {                
-//         if(empty($Brand_ID)) {
-//             echo "<font color='red'>ID field is empty.</font><br/>";
-//         }
-        
-//         if(empty($Brand_Name)) {
-//             echo "<font color='red'>Brand name field is empty.</font><br/>";
-//         }
-        
-//         if(empty($Brand_Desc)) {
-//             echo "<font color='red'>Brand description field is empty.</font><br/>";
-//         }
-        
-//         //link to the previous page
-//         echo "<br/><a href='brands.php'>Go Back</a>";
-//     } else { 
-//         // if all the fields are filled (not empty)             
-//         //insert data to database
-//         $result = mysqli_query($mysqli, "INSERT INTO BRAND(Brand_ID, Brand_Name, Brand_Desc) VALUES('$Brand_ID','$Brand_Name','$Brand_Desc')");
-        
-//         //redirect to brands list
-//          header("Location: brands.php");
-//     }
-// }
-// //update brand information in db
-// if(isset($_POST['update']))
-// {    
-//     $id = $_POST['id'];
-//     $Brand_ID = $_POST['Brand_ID'];
-//     $Brand_Name = $_POST['Brand_Name'];
-//     $Brand_Desc = $_POST['Brand_Desc'];    
-    
-//     // Check for empty fields
-//     if(empty($Brand_ID) || empty($Brand_Name) || empty($Brand_Desc)) {            
-//         if(empty($Brand_Name)) {
-//             echo "<font color='red'>Brand name field is empty.</font><br/>";
-//         }
-        
-//         if(empty($Brand_Desc)) {
-//             echo "<font color='red'>Brand description field is empty.</font><br/>";
-//         }
-            
-//     } else {    
-//         //Update Query
-//         $query = "UPDATE BRAND SET Brand_ID='$Brand_ID' Brand_Name='$Brand_Name', Brand_Desc='$Brand_Desc' WHERE id=$id";
-//         $result = mysqli_query($connection, $query);
-//        // print_r($result);
-        
-        
-//         //redirecting to brands list
-//         header("Location: brands.php");
-//     }
-// }
-// $id = $_GET['id'];
- 
-// //selecting data associated with this particular id
-// $result = mysqli_query($connection, "SELECT * FROM BRAND WHERE id=$id");
-//  print_r($result);
-// while($res = mysqli_fetch_array($result))
-// {
-//     $Brand_Name = $res['Brand_Name'];
-//     $Brand_Desc = $res['Brand_Desc'];
-// }
+    }
+elseif($_SERVER['REQUEST_METHOD'] == 'GET') {
+    //QUERY THE DATABASE AND STORE ALL USERS INTO A VARIABLE
+    //print $_GET["id"];
+    $get_id = $_GET["id"];
+    // Create your query
+    $query = "SELECT * FROM BRAND 
+              WHERE Brand_ID = $get_id";
+    //print $query;
+    // Run your query
+    $result_update = mysqli_query($connection, $query);
+
+    // Check if the database returned anything
+    if($result_update) {
+        $brands = mysqli_fetch_all($result_update, MYSQLI_ASSOC);    
```

Comment:
I would probably change this to a mysqli_fetch_assoc since when you select from the BRAND table by ID, you're only going to be expecting a single result. You could then also remove the foreach below since we're only retrieving a single brand.